kbase-extension/static/kbase/js/widgets/function_output/modeling/kbaseTabTable.js

```diff
@@ -175,9 +176,9 @@ define([
 
                 // note: must add table first
                 tabPane.append(
-                    '<table class="table table-bordered table-striped" style="margin-left: auto; margin-right: auto;">'
+                    '<table class="table table-bordered table-striped" style="width:100%">'
```

Comment:
This was necessary as it's being rendered while hidden, and DataTables will stuff `width: 0px` in there and squish the table. It needs to have `width: X%` to avoid that: https://datatables.net/examples/basic_init/flexible_width.html

Reply:
yep, I have been stung by that before!